marklogic-data-hub-central/ui/src/components/header/system-info.tsx

```diff
@@ -55,48 +77,58 @@ const SystemInfo = (props) => {
+                </div>
+                <div className={styles.cardsContainer}>
+                    <div className={styles.cards}>
+                        <Row gutter={16} type="flex" >
 
-                        <Col>
-                            <Card size="small" className={styles.download} >
-                                <div className={styles.title}>Download Configuration Files</div>
-                                <p>Download a zip file containing flow definitions, step definitions and other user artifacts created or modified by Hub Central.</p>
-                                <div className={styles.buttonContainer}>
-                                    <Button 
-                                        type="primary" 
-                                        disabled = {! authorityService.canDownloadProjectFiles()}
-                                        onClick={download}
-                                    >Download</Button>
-                                </div>
-                            </Card>
-                        </Col>
+                            <Col>
+                                <Card size="small" className={styles.download} >
+                                    <div className={styles.title}>Download Configuration Files</div>
+                                    <p>Download a zip file containing flow definitions, step definitions and other user artifacts created or modified by Hub Central.</p>
+                                    <div className={styles.buttonContainer}>
+                                        <Button
+                                            type="primary"
+                                            disabled = {! authorityService.canDownloadProjectFiles()}
+                                            onClick={download}
+                                        >Download</Button>
+                                    </div>
+                                </Card>
+                            </Col>
 
-                        <Col>
-                            <Card size="small" className={styles.clearAll} >
-                                <div className={styles.title}>Clear All User Data</div>
-                                <p>Delete all user data in STAGING, FINAL, and JOBS databases.</p>
-                                <div className={styles.buttonContainer}>
-                                    <Button type="primary">Clear</Button>
-                                </div>
-                            </Card>
-                        </Col>
+                            <Col>
+                                <Card size="small" className={styles.clearAll}>
+                                    {isLoading === true ? <div className={styles.spinRunning}>
+                                         <Spin size={"large"} />
+                                    </div>: ''}
+                                    <div className={styles.title} data-testid="clearData">Clear All User Data</div>
```

Comment:
If getByText('Clear All User Data') works we can get rid of the data-testid here. data-testid should be used as a last resort. RTL encourages good testing practices with the primary guiding principle:
`The more your tests resemble the way your software is used, the more confidence they can give you.`

Reply:
@AkshayBajajML ,
getByText('Clear All User Data') doesn't work ?

Reply:
Since Alert "Clear All User Data completed successfully" has initial words same as "Clear All User Data" button, due to this most of the test cases fail so had to use data-testid in order to avoid ambiguity.

Reply:
But, doesn't the alert appears after the button is clicked ? So, before the button is clicked, getByText('Clear All User Data') refers to the button and there is no ambiguity right ?

Reply:
As Srikanth previously mentioned that alert is set to none when button is not clicked due to which its still present in DOM even it is not visible hence fails the test case using getByText.